std/heap.zig

```diff
@@ -34,28 +34,17 @@ fn cShrink(self: *Allocator, old_mem: []u8, old_align: u29, new_size: usize, new
 /// Thread-safe and lock-free.
 pub const DirectAllocator = struct {
     allocator: Allocator,
-    heap_handle: ?HeapHandle,
```

Comment:
Yes, and similarly it's possible to also make a global `HeapAllocator` for windows, that would use `GetProcessHeap()` instead of creating a new one. There are some other changes I would like to make on this, I'll probably write a PR sometime. https://docs.microsoft.com/en-gb/windows/desktop/api/heapapi/nf-heapapi-getprocessheap